brave/brave-ios
# Fix #3975: Tab Tray refactor.

## Summary of Changes


This pull request fixes #3975 

## Submitter Checklist:


## Test Plan:

Verify that tab tray actions work with the new implementation, test stuff like:
- switching between private and regular modes
- add new tab, remove tab
- reorder tabs
- remove tab by swiping left or right

## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
